Bio/GenBank/__init__.py

```diff
@@ -1194,7 +1194,12 @@ def feature_qualifier(self, key, value):
             # otherwise just skip this key
             return
 
-        value = value.replace('"', '')
+        # keep GenBank's escaping requirement "" while removing single quotes "
+        # remove empty values (unique case of double quotes)
+        value = re.sub('^""$', '', value)
+        # remove single quotes unless for escaping purposes ""
+        value = value.replace('""', '~^').replace('"', '').replace('~^', '""')
```

Comment:
Won't this break if ``value`` happens to have ``~^`` in it already? A simple if/else would be clearer I think.